src/main/java/games/strategy/engine/framework/startup/ui/SetupPanel.java

```diff
@@ -168,7 +159,6 @@ void layoutPlayerComponents(JPanel panel, List<PlayerSelectorRow> playerRows, Ga
     setAllTypes.setAction(setAllTypesAction);
 
     panel.validate();
-    panel.invalidate();
```

Comment:
This `invalidate()` call is superfluous because the call to `Container#removeAll()` on the very first line of this method already invalidates the panel.  In fact, it's probably incorrect to call `invalidate()` **after** `validate()`; I suspect it should have been the other way around if invalidation was actually necessary.